zeebe/broker/src/main/java/io/camunda/zeebe/broker/bootstrap/PartitionManagerStep.java

```diff
@@ -7,11 +7,19 @@
  */
 package io.camunda.zeebe.broker.bootstrap;
 
+import io.atomix.cluster.MemberId;
+import io.camunda.zeebe.broker.Loggers;
+import io.camunda.zeebe.broker.SpringBrokerBridge;
 import io.camunda.zeebe.broker.partitioning.PartitionManagerImpl;
 import io.camunda.zeebe.scheduler.ConcurrencyControl;
 import io.camunda.zeebe.scheduler.future.ActorFuture;
+import io.camunda.zeebe.topology.state.ClusterTopology;
+import org.slf4j.Logger;
 
 final class PartitionManagerStep extends AbstractBrokerStartupStep {
+  private static final Logger LOGGER = Loggers.SYSTEM_LOGGER;
+  private static final int ERROR_CODE_ON_INCONSISTENT_TOPOLOGY = 3;
```

Comment:
:question: Any particular reason for using `3`? In `VirtualMachineErrorHandler` we use 156 (ascii code Z + B)

Reply:
Some exit codes have [commonly used meaning](https://tldp.org/LDP/abs/html/exitcodes.html). 3 is not defined in that sense. So no other reason. What is the significance of 156? Does it makes sense to re-use it here?

Reply:
I wouldn't reuse it. I'm fine with using 3, I was just curious